src/redux/books/books.js

```diff
@@ -6,22 +6,22 @@ const books = [];
 const BookReducer = (state = books, action) => {
   switch (action.type) {
```

Comment:
- [ ] Please from the project requirement, You should define `initialState` that contains some books. like this :point_down: 
  ```javascript
  const initialState = {
    books: [
      {
        item_id: "item1",
        title: "The Great Gatsby",
        author: "John Smith",
        category: "Fiction",
      },
      {
        item_id: "item2",
        title: "Anna Karenina",
        author: "Leo Tolstoy",
        category: "Fiction",
      },
      {
        item_id: "item3",
        title: "The Selfish Gene",
        author: "Richard Dawkins",
        category: "Nonfiction",
      },
    ],
  }
  ```